hs/src/Reach/Connector/ALGO.hs

```diff
@@ -373,7 +374,8 @@ talloc tk = do
   liftIO $ modifyIORef eTxnsR $ (+) 1
   txni <- liftIO $ readIORef eTxnsR
   when (txni >= algoMaxTxGroupSize) $ do
-    bad "too many txns"
+    bad $ texty $
+      "Exceeded the maximum size of an atomic transfer group: " <> show algoMaxTxGroupSize
```

Comment:
If we want to be helpful, we could say "This is caused by too many transfers in one atomic step."